core/ui/PageControls/layout.tid

```diff
@@ -0,0 +1,23 @@
+title: $:/core/ui/Buttons/layout
+tags: $:/tags/PageControls
+caption: {{$:/core/images/layout-button}} {{$:/language/Buttons/LayoutSwitcher/Caption}}
+description: {{$:/language/LayoutSwitcher/Description}}
+list-after: $:/core/ui/Buttons/theme
+
+\whitespace trim
+\define control-panel-button(class)
+\whitespace trim
+<$button tooltip={{$:/language/Buttons/LayoutSwitcher/Hint}} aria-label={{$:/language/Buttons/LayoutSwitcher/Caption}} class="""$(tv-config-toolbar-class)$ $class$""">
+{{$:/core/ui/KeyboardShortcuts/switcher}}
```

Comment:
I think it would be clearer not to use a transclusion here, but instead include the action-sendmessage widget directly.

Reply:
Ok, I thought we should do abstractions like this. Maybe literal programming is different from traditional programming here.